platform/replication/core/src/main/java/com/peregrine/admin/replication/impl/DistributionReplicationService.java

```diff
@@ -243,8 +242,4 @@ private void setup(Configuration configuration) {
         return answer;
     }
 
-    @Override
-    public List<Resource> prepare(Collection<Resource> resourceList) throws ReplicationException {
-        return new ArrayList<>(resourceList);
-    }
```

Comment:
I do not like depending on the default implementation. As we saw, they get changed for whatever reason, and ppl don't check how it affects the other use cases.